react/features/analytics/reducer.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import { ReducerRegistry } from '../base/redux';
+
+import { UPDATE_LOCAL_TRACKS_DURATION } from './actionTypes';
+
+/**
+ * Initial state.
+ */
+const DEFAULT_STATE = {
+    localTracksDuration: {
+        audio: {
+            startedTime: -1,
+            value: 0
+        },
+        video: {
+            camera: {
+                startedTime: -1,
+                value: 0
+            },
+            desktop: {
+                startedTime: -1,
+                value: 0
+            }
+        },
+        conference: {
+            startedTime: -1,
+            value: 0
+        }
+    }
+};
```

Comment:
Why not track this in lib-jitsi-meet?

Reply:
Honestly I haven't thought about it. I think the implementation would be a little bit more complicated there.

Do you think it fits there better? What are the benefits?

Do you want me to move the implementation there?

Reply:
I want you to at least think about other implementations yes and then decide what you believe is better. If models exist already to encapsulate knowledge, why not consider keeping that knowledge in there? There are already some existing object in lib-jitsi-meet that track analytics too.

Reply:
I haven't said there isn't analytics in lib-jitsi-meet and I'm well aware of that. 

Not considering implementing it in lib-jitsi-meet is my mistake indeed.

In this case I don't see huge benefits of moving it to lib-jitsi-meet and in the same time I don't mind moving it if you’d prefer it.

IMHO generally I'd be in favour of removing the analytics from lib-jitsi-meet in order to simplify the whole analytics implementation and interfaces. Although I'm not sure if we can move every analytics event out of lib-jitsi-meet.